Agreements/Internal/Rat_Ant.txt

```diff
@@ -0,0 +1 @@
+COPX:*:*:NELBO:EIDW:*:Rathlin:Antrim:*:26000:NELBO
```

Comment:
Just like you've made these changes in this pull request, is it possible to edit the changelog in the same one too? :))